ssh_key_test.go

```diff
@@ -0,0 +1,61 @@
+package digitalocean
+
+import (
+	. "github.com/motain/gocheck"
+	"testing"
+)
+
+func TestSSHKey(t *testing.T) {
+	TestingT(t)
+}
+
+func (s *S) Test_CreateSSHKey(c *C) {
+	testServer.Response(202, nil, sshKeyExample)
+
+	opts := CreateSSHKey{
+		Name:      "A",
+		PublicKey: "abcd",
+	}
+
+	id, err := s.client.CreateSSHKey(&opts)
+
+	req := testServer.WaitRequest()
+
+	c.Assert(req.Form["name"], DeepEquals, []string{"A"})
```

Comment:
I'm thinking this will fail against master. I've recently moved away from sending form params and instead sending JSON body. The test server doesn't yet decode the JSON response, so you can't currently assert JSON content, but this will likely fail as is.